rag/utils/base64_image.py

```
@@ -40,55 +40,47 @@ async def image2id(d: dict, storage_put_func: partial, objname: str, bucket: str
-                        img.close()
-                    except Exception:
-                        pass
+        except (OSError, ValueError) as e:
+            logging.warning(f"Saving image exception: {e}")
+            return None
+        finally:
+            for owned_img in owned_images:
+                try:
+                    owned_img.close()
+                except Exception:
+                    pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)